File: code/1005/1005.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const num1 = parseFloat(lines[0]);
const num2 = parseFloat(lines[1]);

const num1_weight = 3.5
const num2_weight = 7.5

const average = (num1 * num1_weight + num2 * num2_weight) / (num1_weight + num2_weight);

const roundNumber = ((Math.round(average * 100000)) / 100000).toString().split('.');

console.log(`MEDIA = ${roundNumber[0]}.${(roundNumber[1] || '').padEnd(5, "0")}`);

File: code/1005/1005.rb
input_num1 = gets
input_num2 = gets

num1 = input_num1.to_f
num2 = input_num2.to_f

num1_weight = 3.5
num2_weight = 7.5

average = (num1 * num1_weight + num2 * num2_weight) / (num1_weight + num2_weight)

puts "MEDIA = #{'%.5f' % average.round(5)}"
